.storybook/main.js

```diff
@@ -0,0 +1,32 @@
+// your app's webpack.config.js
+const custom = require("../webpack.config.js")();
+
+module.exports = {
+  "stories": [
+    "../src/**/*.stories.mdx",
+    "../src/**/*.stories.@(js|jsx|ts|tsx)"
+  ],
+  "addons": [
+    "@storybook/addon-links",
+    "@storybook/addon-essentials"
```

Comment:
I recommend these two awesome plugins:

```
    "@storybook/addon-controls",
    "@storybook/addon-docs",
```

Probably included in essentials? Not sure.

Reply:
yes, they are included